quesma/quesma/functionality/bulk/bulk.go

```diff
@@ -261,74 +261,71 @@ func sendToClickhouse(ctx context.Context, clickhouseDocumentsToInsert map[strin
 	for indexName, documents := range clickhouseDocumentsToInsert {
 		phoneHomeAgent.IngestCounters().Add(indexName, int64(len(documents)))
 
-		config.RunConfiguredIngest(ctx, cfg, indexName, make(types.JSON), func() error {
```

Comment:
The code no longer uses `RunConfiguredIngest` since that function had it's own logic of checking if an index is enabled for ingest based on the configuration, but now it's handled by table_resolver instead earlier.